sebs/faas/system.py

```diff
@@ -115,7 +115,7 @@ def package_code(
         pass
 
     @abstractmethod
-    def create_function(self, code_package: Benchmark, func_name: str) -> Function:
+    def create_function(self, code_package: Benchmark, func_name: str, num: int) -> Function:
```

Comment:
We shouldn't change this API

Reply:
I had added the `num` parameter to know how many containers to dispatch.